src/components/bottom-drawer/index.tsx

```diff
@@ -0,0 +1,47 @@
+import useBreakPoint from '@/hooks/useBreakPoint'
+import { Button, Dialog, DialogBody, DialogContent, DialogOverlay } from 'gfx-component-lib'
+
+export const BottomDrawer = ({
+  isOpen,
+  setOpen,
+  children
+}: {
+  isOpen: boolean
+  setOpen: (b: boolean) => void
+  children: React.ReactNode
+}) => {
+  const breakpoint = useBreakPoint()
+  const isMobile = breakpoint.isMobile
+  return (
+    <Dialog
+      open={isOpen}
+      onOpenChange={(open) => {
+        setOpen(open)
+      }}
+    >
+      <DialogOverlay />
+      <DialogContent
+        className={`w-full h-max max-h-[100dvh] overflow-y-scroll rounded-b-none 
```

Comment:
I noticed in a couple areas we are starting to use w-full for modals, do you think we should add a new prop e.g fullWidth which adds w-full automatically?